storycraftr/llamaindex/cli.py

```
@@ -0,0 +1,236 @@
+import click
+from pathlib import Path
+from typing import List, Dict, Optional
+
+from rich.console import Console
+from rich.panel import Panel
+from rich.markdown import Markdown
+from rich.table import Table
+
+from storycraftr.llamaindex.core import (
+    build_index,
+    load_index,
```

Ruff (0.8.2)

`storycraftr.llamaindex.core.load_index` imported but unused

Remove unused import

(F401)